handsontable/src/plugins/dialog/dialog.js

```diff
@@ -40,7 +48,6 @@ const SHORTCUTS_CONTEXT_NAME = `plugin:${PLUGIN_KEY}`;
  *   customClassName: 'custom-dialog',
  *   background: 'semi-transparent',
  *   contentBackground: false,
- *   contentDirections: 'column',
  *   animation: false,
  *   closable: true,
```

Comment:
I'd add `a11y` options here to show the full spectrum of available options for that plugin.